flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -831,6 +896,96 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         this.inOut = data;
     }
 
+    clearFocus() {
+        this.inFocus = false;
+        Coherent.trigger('UNFOCUS_INPUT_FIELD');
+        this._inOutElement.style = null;
+        this.getChildById("header").style = null;
+        if (this.check_focus) {
+            clearInterval(this.check_focus);
+        }
+    }
+
+    initKeyboardScratchpad() {
+        window.document.addEventListener('click', () => {
+
+            const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "NONE");
+            const mcduTimeout = parseInt(NXDataStore.get("CONFIG_MCDU_KB_TIMEOUT", "60"));
+            const isPoweredL = SimVar.GetSimVarValue("L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED", "Number");
+            const isPoweredR = SimVar.GetSimVarValue("L:A32NX_ELEC_AC_2_BUS_IS_POWERED", "Number");
+
+            // TODO: L/R MCDU
+            if (mcduInput != "NONE") {
```

Comment:
Use strict compare instead (if possible)

e.g.

```suggestion
            if (mcduInput !== "NONE") {
```

Reply:
Noted, also changing to === true rather than !== false